code/game/gamemodes/wizard/magic_tarot.dm

```diff
@@ -0,0 +1,885 @@
+
+/datum/tarot/the_fool
+	name = "0 - The Fool"
+	desc = "Where journey begins."
+	card_icon = "the_fool"
+
+/datum/tarot/the_fool/activate(mob/living/target)
+	target.forceMove(pick(GLOB.latejoin))
+	to_chat(target, "<span class='userdanger'>You are abruptly pulled through space!</span>")
+
+/datum/tarot/the_magician
+	name = "I - The Magician"
+	desc = "May you never miss your goal."
+	card_icon = "the_magician"
+
+/datum/tarot/the_magician/activate(mob/living/target)
+	target.apply_status_effect(STATUS_EFFECT_BADASS)
+	to_chat(target, "<span class='notice'>You feel badass.</span>")
+
+/datum/tarot/the_high_priestess
+	name = "II - The High Priestess"
+	desc = "Mother is watching you."
+	card_icon = "the_high_priestess"
+
+/datum/tarot/the_high_priestess/activate(mob/living/target)
+	new /obj/effect/abstract/bubblegum_rend_helper(get_turf(target), target, 20)
+
+/obj/effect/abstract/bubblegum_rend_helper
+	name = "bubblegum_rend_helper"
+
+/obj/effect/abstract/bubblegum_rend_helper/Initialize(mapload, mob/living/owner, damage)
+	. = ..()
+	INVOKE_ASYNC(src, PROC_REF(rend), owner, damage)
+
+/obj/effect/abstract/bubblegum_rend_helper/proc/rend(mob/living/owner, damage)
+	if(!owner)
+		for(var/mob/living/L in shuffle(view(9, src)))
+			owner = L
+			break
+	owner.Immobilize(3 SECONDS)
+	for(var/i in 1 to 3)
+		var/turf/first_turf = get_turf(owner)
+		new /obj/effect/decal/cleanable/blood/bubblegum(first_turf)
+		if(prob(50))
+			new /obj/effect/temp_visual/bubblegum_hands/rightsmack(first_turf)
+		else
+			new /obj/effect/temp_visual/bubblegum_hands/leftsmack(first_turf)
+		sleep(6)
+		var/turf/second_turf = get_turf(owner)
+		to_chat(owner, "<span class='userdanger'>Something huge rends you!</span>")
+		playsound(second_turf, 'sound/misc/demon_attack1.ogg', 100, TRUE, -1)
+		owner.adjustBruteLoss(damage)
+	qdel(src)
+
+/datum/tarot/the_empress
+	name = "III - The Empress"
+	desc = "May your rage bring power."
+	card_icon = "the_empress"
+
+/datum/tarot/the_empress/activate(mob/living/target)
+	if(ishuman(target))
+		var/mob/living/carbon/human/H = target
+		H.reagents.add_reagent("mephedrone", 4.5)
+		H.reagents.add_reagent("mitocholide", 12)
+
+/datum/tarot/the_emperor
+	name = "IV - The Emperor"
+	desc = "Challenge me!"
+	card_icon = "the_emperor"
+
+/datum/tarot/the_emperor/activate(mob/living/target)
+	var/list/L = list()
+	for(var/turf/T in get_area_turfs(/area/station/command/bridge))
+		if(is_blocked_turf(T))
+			continue
+		L.Add(T)
+
+	if(!length(L))
+		to_chat(target, "<span class='warning'>Huh. No bridge? Well, that sucks.</span>")
+		return
+
+	target.forceMove(pick(L))
+	to_chat(target, "<span class='userdanger'>You are abruptly pulled through space!</span>")
+
+/datum/tarot/the_hierophant
+	name = "V - The Hierophant"
+	desc = "Two prayers for the lost."
+	card_icon = "the_hierophant"
+
+/datum/tarot/the_hierophant/activate(mob/living/target)
+	if(ishuman(target))
+		var/mob/living/carbon/human/H = target
+		if(H.wear_suit)
```

Comment:
Could you make both of these an early return?

Reply:
done